it/uk/gov/hmrc/emcstfe/support/IntegrationBaseSpec.scala

```diff
@@ -34,7 +34,8 @@ trait IntegrationBaseSpec extends UnitSpec with WireMockHelper with GuiceOneServ
   def servicesConfig: Map[String, _] = Map(
     "microservice.services.auth.port" -> WireMockHelper.wireMockPort,
     "microservice.services.chris.port" -> WireMockHelper.wireMockPort,
-    "auditing.consumer.baseUri.port" -> WireMockHelper.wireMockPort
+    "auditing.consumer.baseUri.port" -> WireMockHelper.wireMockPort,
+    "play.http.router" -> "testOnlyDoNotUseInAppConf.Routes"
```

Comment:
Default router is prod.Routes (which doesn't include the test-only routes) so we need to make the default router testOnlyDoNotUseInAppConf.Routes explicitly in the IT config.